src/core/file.cc

```diff
@@ -204,7 +204,8 @@ posix_file_impl::flush() noexcept {
 
 future<struct stat>
 posix_file_impl::stat() noexcept {
```

Comment:
> core: add label to io_threaded_fallbacks to categorize operations

Please add the description to the patch commitlog so it is immortalized in the repository.

Reply:
Done!